app/page.tsx

```diff
@@ -29,8 +46,94 @@ export default function LandingPage() {
+          className="absolute inset-0 top-pr-60 object-cover mo:hidden ta:hidden"
+        />
+        <Image
+          src="/images/landing/img-Landing-bg-ta.png"
+          alt="background-tablet"
+          fill
+          className="absolute inset-0 top-pr-60 hidden object-cover mo:hidden ta:block"
+        />
+        <div
+          className="absolute inset-0 top-pr-60 hidden bg-cover bg-center bg-no-repeat mo:block"
+          style={{
+            backgroundImage: "url('/images/landing/img-Landing-bg-mo.png')",
+          }}
+        />
+
+        <div className="absolute left-1/2 top-pr-204 flex -translate-x-1/2 flex-col items-center justify-center gap-pr-20 text-center mo:top-pr-175 ta:top-pr-220">
+          <h2 className="flex gap-pr-24 whitespace-nowrap text-48sb text-t-primary mo:gap-pr-4 mo:text-24sb ta:text-40sb">
+            함께 만들어가는 투두 리스트
+            <Image
+              src="/images/landing/img-Tool.svg"
+              alt="Tool"
+              width={56}
+              height={56}
+              className="mo:size-pr-28 ta:size-pr-48"
+            />
+          </h2>
+          <h2 className="whitespace-nowrap bg-gradient-to-r from-brand-primary to-brand-tertiary bg-clip-text text-center text-64sb text-transparent mo:text-32sb ta:text-48sb">
+            Coworkers
+          </h2>
+        </div>
+
+        <div className="absolute bottom-pr-120 left-1/2 -translate-x-1/2 mo:bottom-pr-48 ta:bottom-pr-119">
+          <Buttons
+            text="지금 시작하기"
+            backgroundColor="gradient"
+            size="M"
+            className="w-pr-373 whitespace-nowrap mo:w-pr-343"
+            rounded={true}
+            onClick={handleButtonClick}
+          />
+        </div>
+      </section>
+
+      <section className="relative mt-pr-60 flex flex-col items-center gap-pr-80 tamo:gap-pr-24">
+        <LandingCardItemsFirst />
+        <LandingCardItemsSecond />
+        <LandingCardItemsthird />
+      </section>
+
+      <footer>
+        <div className="relative flex h-pr-1080 w-screen flex-col items-center">
+          <Image
+            src="/images/landing/img-Landing-bottom.png"
+            alt="background-bottom"
+            fill
+            className="absolute inset-0 object-cover mo:hidden ta:hidden"
+          />
+          <Image
+            src="/images/landing/img-Landing-bottom-ta.png"
+            alt="background-bottom-tablet"
+            fill
+            className="absolute inset-0 hidden object-cover mo:hidden ta:block"
+          />
+          <div
+            className="absolute inset-0 hidden bg-cover bg-center bg-no-repeat mo:block"
+            style={{
+              backgroundImage:
+                "url('/images/landing/img-Landing-bottom-mo.png')",
+            }}
+          />
+
+          <div className="absolute top-pr-230 flex flex-col items-center mo:top-pr-123 ta:top-pr-176">
+            <h1 className="mb-pr-24 text-center text-40sb text-t-primary mo:text-24sb ta:text-40sb">
```

Comment:
여기도 pc와 태블릿이 동일한 폰트 사이즈를 가지고 있다면, 태블릿에서 ta:text-40sb는 지우셔도 될 것 같습니다~
```suggestion
            <h1 className="mb-pr-24 text-center text-40sb text-t-primary mo:text-24sb">
```

Reply:
수정했습니다